test/test_transformers.py

```diff
@@ -66,6 +66,50 @@ def get_rtol(true_value: torch.Tensor, computed_value: torch.Tensor) -> float:
     torch.nan_to_num_(deviation, nan=default_rtol[computed_value.dtype])
     return deviation.max().item()
 
+
+backend_map = {
+    SDPBackend.MATH: {"enable_math": True, "enable_flash": False, "enable_mem_efficient": False},
+    SDPBackend.FLASH_ATTENTION: {"enable_math": False, "enable_flash": True, "enable_mem_efficient": False},
+    SDPBackend.EFFICIENT_ATTENTION: {
+        "enable_math": False, "enable_flash": False, "enable_mem_efficient": True}
+}
+
+
+def rand_tensor(shape: Tuple[Union[int, List[int]]], device: str, dtype: torch.dtype, type: str,
```

Comment:
The name should actually probably be a little different than make_tensor and really be like `rand_sdpa_tensor` since it makes a 4 dim batch_size, num_heads, seq_len, head_dim formatted tensor